code/game/sound.dm

```diff
@@ -151,9 +156,6 @@
 			sound_to_use.volume *= pressure_factor
 			//End Atmosphere affecting sound
 
-		if(sound_to_use.volume <= 0)
-			return //No sound
-
 		var/dx = turf_source.x - turf_loc.x // Hearing from the right/left
```

Comment:
You cannot remove this line. Pressure can reduce a sound volume to zero.

Reply:
I moved the volume check to a different scope to calculate audible range. I completely forgot about pressure decreasing volume.

Re-added via https://github.com/tgstation/tgstation/pull/88517/commits/0b42bdfb32d51c2a0408b4cee753e8dfecafb9e7